src/dispatch/plugins/dispatch_slack/workflow.py

```diff
@@ -0,0 +1,278 @@
+
+    blocks = [Section(text="*Workflows*")]
+    for w in workflows:
+        artifact_links = ""
+        for a in w.artifacts:
+            artifact_links += f"- <{a.weblink}|{a.name}> \n"
+
+        blocks.append(
+            Section(
+                fields=[
+                    f"*Name:* \n <{w.weblink}|{w.workflow.name}>"
+                    f"*Workflow Description:* \n {w.workflow.description}"
+                    f"*Run Reason:* \n {w.run_reason}"
+                    f"*Creator:* \n {w.creator.individual.name}"
+                    f"*Status:* \n {w.status}"
+                    f"*Artifacts:* \n {artifact_links}"
+                ]
+            )
+        )
+
+    blocks = Message(blocks=blocks).build()["blocks"]
+    await respond(blocks=blocks, response_type="ephemeral")
+
+
+async def handle_workflow_run_command(
+    ack,
+    body,
+    client,
+    context,
+    db_session,
+):
+    """Handles the workflow run command."""
+    await ack()
+    incident = incident_service.get(db_session=db_session, incident_id=context["subject"].id)
+
+    blocks = [
+        Context(elements=[MarkdownText(text="Select a workflow to run.")]),
+        workflow_select(
+            db_session=db_session, dispatch_action=True, project_id=incident.project.id
+        ),
+    ]
+
+    modal = Modal(
+        title="Run Workflow",
+        blocks=blocks,
+        submit="Run",
+        close="Close",
+        callback_id=RunWorkflowActions.submit,
+        private_metadata=context["subject"].json(),
+    ).build()
+
+    await client.views_open(view=modal, trigger_id=body["trigger_id"])
+
+
+@app.view(
+    RunWorkflowActions.submit,
+    middleware=[action_context_middleware, db_middleware, user_middleware, modal_submit_middleware],
+)
+async def handle_workflow_submission_event(ack, body, client, context, db_session, form_data, user):
+    """Handles workflow submission event."""
+    incident = incident_service.get(db_session=db_session, incident_id=context["subject"].id)
+
+    workflow_id = form_data.get(RunWorkflowBlockIds.workflow_select)["value"]
+    workflow = workflow_service.get(db_session=db_session, workflow_id=workflow_id)
+
+    params = {}
+    named_params = []
+    for i in form_data.keys():
+        if i.startswith(RunWorkflowBlockIds.param_input):
+            key = i.split("-")[1]
+            value = form_data[i]
+            params.update({key: value})
+            named_params.append({"key": key, "value": value})
+
+    creator = participant_service.get_by_incident_id_and_email(
+        db_session=db_session, incident_id=incident.id, email=user.email
+    )
+
+    instance = workflow_service.create_instance(
+        db_session=db_session,
+        instance_in=WorkflowInstanceCreate(
+            workflow=workflow,
+            incident=incident,
+            creator=creator,
+            run_reason=form_data[RunWorkflowBlockIds.reason_input],
+            parameters=named_params,
+        ),
+    )
```

Comment:
```suggestion
    instance = workflow_service.create_instance(
        db_session=db_session,
        workflow=workflow,
        instance_in=WorkflowInstanceCreate(
            incident=incident,
            creator=creator,
            run_reason=form_data[RunWorkflowBlockIds.reason_input],
            parameters=named_params,
        ),
    )
```

Raises -> 

```bash
ERROR:Error: create_instance() missing 1 required keyword-only argument: 'workflow':/Users/wshel/Projects/dispatch/src/dispatch/plugins/dispatch_slack/bolt.py:app_error_handler:40
Traceback (most recent call last):
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_bolt/listener/asyncio_runner.py", line 65, in run
    returned_value = await listener.run_ack_function(request=request, response=response)
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_bolt/listener/async_listener.py", line 119, in run_ack_function
    return await self.ack_function(
  File "/Users/wshel/Projects/dispatch/src/dispatch/plugins/dispatch_slack/workflow.py", line 200, in handle_workflow_submission_event
    instance = workflow_service.create_instance(
TypeError: create_instance() missing 1 required keyword-only argument: 'workflow'
```

This seems like it's broken in main/pre-bolt as well:

https://github.com/Netflix/dispatch/blob/0b75d5d00394f79bde54ab5c0c631cf87b975a29/src/dispatch/plugins/dispatch_slack/modals/workflow/handlers.py#L179-L188

https://github.com/Netflix/dispatch/blob/0b75d5d00394f79bde54ab5c0c631cf87b975a29/tests/workflow/test_workflow_service.py#L57-L68